src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -1968,6 +1970,11 @@ public async Task ValidateOrganizationUserUpdatePermissions(Guid organizationId,
         {
             throw new BadRequestException("Custom users can only grant the same custom permissions that they have.");
         }
+
+        if (FlexibleCollectionsIsEnabled && newType == OrganizationUserType.Manager)
```

Comment:
I'm a little concerned this might block other updates that are unrelated to the role, given that we still have the Manager enum type floating around. I suggest we check that either the orgUser is being created, or that their role is being explicitly changed. e.g. 

```suggestion
        if (FlexibleCollectionsIsEnabled && newType == OrganizationUserType.Manager && (oldType == null) || oldType != OrganizationUserType.Manager)
```

This is probably not necessary but seems safer.